### DISCUSSION TITLE
Light Protanopia & Deuteranopia hover text

### DISCUSSION THREAD

**Discussion by ericpruitt** [community/community#16347] [Accessibility] [answered]
The alt text for the "Light Protanopia & Deuteranopia" option in the themes reads "Light colorblind" instead of the "Light Protanopia & Deuteranopia."

![2022-05-10T18:35:05-0700](https://user-images.githubusercontent.com/481184/167751548-9badbeda-e5b3-4084-bb02-b52893b95133.png)

**Comment by cehfisher** [comment, marked as answer]
Hi @ericpruitt - thank you for providing this feedback! This issue appears to be resolved now. Could you please confirm and close out this issue? Thank you!

<img width="422" alt="Screenshot showing the correct label for the Light Protanopia & Deuteranopia theme." src="https://github.com/community/community/assets/15240149/582ffb52-6782-47e9-8c82-b9e03b5da724">

**Reply by ericpruitt** [reply]
Thanks, fix confirmed.
